src/main/java/org/gingesnap/cdc/connector/DatabaseProvider.java

```diff
@@ -38,6 +41,23 @@ public Properties databaseProperties(Connector connector, Database database) {
          properties.setProperty("publication.autocreate.mode", "filtered");
          properties.setProperty("plugin.name", "pgoutput");
 
+         return properties;
+      }
+   },
+   MSSQL {
+      @Override
+      public Properties databaseProperties(Connector connector, Database database) {
+         Properties properties = new Properties();
+         String databaseName = database.database().orElseThrow(() -> new IllegalStateException("SQL Server requires database name."));
+         properties.setProperty("connector.class", SqlServerConnector.class.getCanonicalName());
+         properties.setProperty(DATABASE_NAMES.name(), databaseName);
+         properties.setProperty("table.include.list", String.format("%s.%s", connector.schema(), connector.table()));
+         properties.setProperty("snapshot.mode", "initial");
+         properties.setProperty("database.encrypt", "false");
```

Comment:
Without this, we have problems connecting to the container. Later we should make all this configurable.